kafka-init/image.yaml

```diff
@@ -0,0 +1,41 @@
+schema_version: 1
+
+name: redhat-amqstreams-1/amqstreams10-kafkainit-openshift
+description: "AMQ Streams base image for Kafka init"
+version: "1.0"
+from: redhat-amqstreams-1/amqstreams10-javabase-openshift:1.0
```

Comment:
where is the javabase image? I cannot see it in the repo.

Reply:
Just added a PR here: https://github.com/jboss-container-images/jboss-amqstreams-1-openshift-image/pull/18